Doc/glossary.rst

```diff
@@ -40,16 +40,18 @@ Glossary
       ABCs with the :mod:`abc` module.
 
    annotation
-      A metadata value associated with a global variable, a class attribute or a
-      function or method parameter or return value, that stores a
-      :term:`type hint`.
+      A label with no defined semantics associated with a variable, a class
+      attribute or a function or method parameter or return value.
 
-      Annotations are stored in the :attr:`__annotations__` special attribute
-      of a module (when annotating a global variable), class (when annotating
-      one of its attributes) or function or method (when annotating a parameter or a
-      return value) and can be accessed using :func:`typing.get_type_hints`.
+      Annotations of local variables cannot be accesed at runtime, but
+      annotations of global variables, class attributes, functions and methods
+      parameters and return values are stored in the :attr:`__annotations__`
```

Comment:
I'd write "and functions" instead of "functions and methods parameters and return values". Or if you really feel that it is worth repeating the "parameters and return values" then at least write "function parameters and return values", i.e. no "s".